mlir/lib/Conversion/ConvertToEmitC/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_mlir_conversion_library(MLIRConvertToEmitC
```

Comment:
You may want to extend the Bazel build configuration similar to
https://github.com/llvm/llvm-project/blob/d7f096e3fe611ae2cc7403c3cf2f88255a47b61d/utils/bazel/llvm-project-overlay/mlir/BUILD.bazel#L7627-L7643

but maybe @mtrofin and @ilovepi can help with that in a follow up.

Reply:
Happy to take care of that in a follow-up!

Reply:
PR #138154